gr-digital/lib/map_bb_impl.h

```diff
@@ -32,7 +32,8 @@ namespace digital {
 class map_bb_impl : public map_bb
 {
 private:
-    unsigned char d_map[0x100];
+    static constexpr size_t d_map_size = 0x100;
```

Comment:
I often get linker errors with static constexpr class members, but I can't quite remember when and how.

Reply:
Googling around it looks like you've maybe encountered [this thing that's been fixed in C++17 (according to one of the answers)](https://stackoverflow.com/questions/8452952/c-linker-error-with-class-static-constexpr).

Shouldn't affect this PR.